homeassistant/components/mqtt/cover.py

```diff
@@ -252,24 +253,10 @@ def _setup_from_config(self, config):
         if tilt_status_template is not None:
             tilt_status_template.hass = self.hass
 
-    async def _subscribe_topics(self):
-        """(Re)Subscribe to topics."""
-        topics = {}
-
-        @callback
-        @log_messages(self.hass, self.entity_id)
-        def tilt_message_received(msg):
-            """Handle tilt updates."""
-            payload = msg.payload
-            tilt_status_template = self._config.get(CONF_TILT_STATUS_TEMPLATE)
-            if tilt_status_template is not None:
-                payload = tilt_status_template.async_render_with_possible_json_value(
-                    payload
-                )
-
-            if not payload.isnumeric():
-                _LOGGER.warning("Payload '%s' is not numeric", payload)
-            elif (
+    def _tilt_message_received(self, payload):
```

Comment:
I think it would be better to name this `_tilt_payload_received` since it is also called from `position_message_received` upon receiving position so it would be clear to understand it is not called only upon tilt message

Reply:
I messed up this PR, as I always do if I have to merge with upstream. I created new PR #50236